frontend/apps/filemanager/filemanager.lua

```diff
@@ -731,7 +731,7 @@ function FileManager:reinit(path, focused_file)
     UIManager:flushSettings()
     self.dimen = Screen:getSize()
     -- backup the root path and path items
-    self.root_path = path or self.file_chooser.path
+    self.root_path = BaseUtil.realpath(path or self.file_chooser.path)
```

Comment:
This isn't necessary in some other places where it does something like `self.(root_)path = path` then?

Reply:
We also do it in
https://github.com/koreader/koreader/blob/526a1fb7276468137045d6227c8f62b58c75daa3/frontend/ui/widget/filechooser.lua#L513-L515
as `FileChooser:changeToPath` can be called from elsewhere.